recipe/meta.yaml

```diff
@@ -32,7 +31,7 @@ test:
 about:
   home: https://github.com/HPAC/matchpy
   license: MIT
-  license_file: LICENSE
+  license_file: LICENSE.txt
```

Comment:
Maybe it's just _LICENSE_, without .txt ?

Reply:
Yes, after my initial confusion I now think that it should just be `LICENSE`. However, that's what it used to be before my last commit to this PR (this one https://github.com/conda-forge/matchpy-feedstock/pull/6/commits/81d4dd39cdba1e44811f8b788ab2c73a2ffb3414), and the build already failed when it was just `LICENSE`.